<launch>
    <arg name="rgb_topic"   default="/camera/color/image_rect_color" />
    <arg name="depth_topic"     default="/camera/aligned_depth_to_color/image_raw" />
    <arg name="pcd_topic"   default="/camera/depth_registered/points" />
    <arg name="queue_size"  default="5" />
    <arg name="hand_pcd_topic"    default="/hand_pcd" />
    <arg name="object_pcd_topic"    default="/object_pcd"/>

    <node name="object_segmentor_node" pkg="seg_hand" type="object_seg_node" respawn="true" >
        <param name="rgb_topic" type="str" value="$(arg rgb_topic)" />
        <param name="depth_topic" type="str" value="$(arg depth_topic)" />
        <param name="pcd_topic" type="str" value="$(arg pcd_topic)" />
        <param name="queue_size" type="int" value="$(arg queue_size)" />
        <param name="hand_pcd_topic" type="str" value="$(arg hand_pcd_topic)" />
        <param name="object_pcd_topic" type="str" value="$(arg object_pcd_topic)" />
    </node>

</launch>